packages/react-intl/src/utils.ts

```diff
@@ -134,3 +134,14 @@ export function getNamedFormat<T extends keyof CustomFormats>(
 
   onError(new UnsupportedFormatterError(`No ${type} format named: ${name}`));
 }
+
+export function unapplyFormatXMLElementFn(
```

Comment:
Can you add a comment here to explain what this function does?

Reply:
Comment added.